cfme/tests/cloud/test_azure_manual.py

```diff
@@ -0,0 +1,241 @@
+import pytest
+
+pytestmark = [pytest.mark.manual]
+
+
+@pytest.mark.manual
+def test_vpc_env_selection():
+    """
+    Test selection of components in environment page of cloud instances
+    with and without selected virtual private cloud
+    Related to BZ 1315945
+
+    Polarion:
+        assignee: anikifor
+        casecomponent: WebUI
+        initialEstimate: 1/2h
+        testSteps:
+            1. Provision an Azure Instance from an Image.
+            2. At the environment page, try to select components without vpc
+            3. At the environment page, try to select components without vpc with vpc
+        expectedResults:
+            1. Instance provisioned and added successfully
+            2. Items are selected successfully
+            3. Items are selected successfully
+    """
+    pass
+
+
+@pytest.mark.manual
+@pytest.mark.tier(2)
+def test_refresh_azure_provider_with_empty_ipv6_config_on_vm():
+    """
+    https://bugzilla.redhat.com/show_bug.cgi?id=1468700
```

Comment:
Please use `Bugzilla:` header in docblock, or `pytest.mark.meta(automates=[BZ(number)])` so that this metadata information can be programatically accessed.